clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -416,7 +426,7 @@ synchronized int numAssignedPartitions() {
                 .collect(Collectors.toList());
     }
 
-    synchronized boolean partitionsAutoAssigned() {
+    public synchronized boolean partitionsAutoAssigned() {
```

Comment:
Called by KafkaConsumer

Reply:
nit: wonder if it's worth changing this name. For example: `hasAutoAssignedPartitions`?

Reply:
ack.